mbs_results/utilities/validation_checks.py

```
@@ -275,6 +275,36 @@ def validate_outlier_detection(df: pd.DataFrame, config: dict):
     df = append_filter_out_questions(df, filtered_questions_path)
     df.to_csv(output_path + outlier_filename, index=False)
 
+def validate_manual_outlier_df(df: pd.DataFrame) -> bool:
+    warnings.warn("A placeholder function for validating ingested manual outliers")
+
+    # Todo: Verbose nested if-else block, should be rewritten to something cleaner
```

I don't think the nested if-else block is needed at all here - maybe you could check the inverse for each condition, so you're only raising an Exception if anything is violated. 

This seems to be more along the lines of how the rest of the validation checks in this script are working, i.e. "if X is violated, raise an error" - rather than "if X isn't violated, return True, else, return an error". 

That should hopefully simplify things a bit more. I don't think there's any reason to return True at all here, if an exception is triggered it will automatically return the exception and stop running.